.env

```diff
@@ -16,7 +16,7 @@ _APP_DB_PORT=3306
 _APP_DB_SCHEMA=appwrite
 _APP_DB_USER=user
 _APP_DB_PASS=password
-_APP_STORAGE_ANTIVIRUS=enabled
```

Comment:
If we disable it for dev, we would need to comment it out in the future 👍🏻

Reply:
Good point, maybe we should do this now. As currently ClamAV servers still have issues.